composer.json

```diff
@@ -14,7 +14,8 @@
     ],
 
     "require": {
-        "php": ">=5.3.1"
+        "php": "^7.4",
```

Comment:
this is a blocker to me. The next minor version of Behat supports 7.2+, so we cannot be stricter in the Gherkin parser

Reply:
As @ciaranmcnulty commented https://github.com/Behat/Gherkin/pull/177#issuecomment-658926614 this parser is very much experimental. And it will be a while before Parsica itself is stable enough to become the default parser for Behat. So I don't think we need to worry about versions now.